.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged
```

Comment:
Please can we add this back in as it only lints ts & tsx files (which is the intention) :)